YourMovieDB/Data/Model/Movie.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Movie.swift
+//  YourMovieDB
+//
+//  Created by Tung Tran on 11/6/19.
+//  Copyright © 2019 Sun. All rights reserved.
+//
+
+import Foundation
+import ObjectMapper
+
+struct Movie: BaseModel {
+    var popularity = 0.0
+    var voteCount = 0
+    var video = false
+    var posterPath = ""
+    var id = 0
+    var adult = false
+    var backdropPath = ""
+    var originalLanguage = ""
+    var originalTitle = ""
+    var genreId = [Int]()
+    var title = ""
+    var voteAverage = 0.0
+    var overView = ""
+    var releaseDate = ""
```

Comment:
```suggestion
    var releaseDate: Date?
```
date nể để dạng optional